agent/proto/agent.proto

```diff
@@ -264,6 +266,29 @@ message BannerConfig {
 	string background_color = 3;
 }
 
+message WorkspaceAgentScriptCompletedRequest {
+    Timing timing = 1;
+}
+
+message WorkspaceAgentScriptCompletedResponse {
+}
+
+message Timing {
+    bytes script_id = 1;
+    string display_name = 2;
+    google.protobuf.Timestamp start = 3;
+    google.protobuf.Timestamp end = 4;
+    int32 exit_code = 5;
+
+    enum Stage {
+        START = 0;
+        STOP = 1;
+        CRON = 2;
```

Comment:
I feel there's a slight mismatch in naming here as I have trouble fitting _cron_ under _stage_, but I also don't have a better suggestion 😅, just an observation.

Reply:
Yeah it does feel a little strange. Couldn't think of a better way of handling this though.